src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,47 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
 
+    public static final String DATE_PATTERN = "dd.MM.yyyy";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_PATTERN);
+        LocalDate parseDateFrom = LocalDate.parse(dateFrom, formatter);
+        formatter.format(parseDateFrom);
```

Comment:
What does this method do?

Reply:
this method parse my "dateFrom" from String to object LocalDate